recipes/podman-compose/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "podman-compose" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/podman-compose-{{ version }}.tar.gz
+    sha256: fb229362f188980ea3fbdee2a25d0a2dd6a0b886d925d5213e22e09f1062ebe9
+  - url: https://raw.githubusercontent.com/containers/podman-compose/v{{ version }}/LICENSE
+    sha256: 8177f97513213526df2cf6184d8ff986c675afb514d4e68a404010521b880643
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  entry_points:
+    - podman-compose = podman_compose:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - pyyaml
+  run:
+    - python >=3.5
+    - pyyaml
+
+test:
+  imports:
+    - podman_compose
+  commands:
+    - python -m pip check
+    - podman-compose --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/containers/podman-compose
+  summary: A script to run docker-compose.yml using podman
+  license: GPL-2.0-or-later
+  license_file: LICENSE
```

Comment:
I opened a PR to include the license file in the source distribution:
https://github.com/containers/podman-compose/pull/229

Maybe add a comment above the license file line as a reminder to check if it's included in a future release:

```yaml
...
# License file pulled in from separate source. See: https://github.com/containers/podman-compose/pull/229
license_file: LICENSE
```

Reply:
Thanks -- I'll merge as soon as the CI finishes.